{"sections":[],"hierarchy":{"paths":[["doc:\/\/com.github.pulse\/documentation\/Pulse","doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger","doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError"]]},"schemaVersion":{"patch":0,"major":0,"minor":3},"abstract":[{"text":"Inherited from ","type":"text"},{"type":"codeVoice","code":"LocalizedError.errorDescription"},{"type":"text","text":"."}],"identifier":{"url":"doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError\/errorDescription","interfaceLanguage":"swift"},"metadata":{"externalID":"s:5Pulse12RemoteLoggerC15ConnectionErrorO16errorDescriptionSSSgvp","symbolKind":"property","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"?"}],"role":"symbol","modules":[{"name":"Pulse"}],"title":"errorDescription","roleHeading":"Instance Property"},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["iOS"],"tokens":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":"? { ","kind":"text"},{"text":"get","kind":"keyword"},{"kind":"text","text":" }"}]}]}],"variants":[{"paths":["\/documentation\/pulse\/remotelogger\/connectionerror\/errordescription"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"symbol","references":{"doc://com.github.pulse/documentation/Pulse":{"role":"collection","url":"\/documentation\/pulse","kind":"symbol","title":"Pulse","abstract":[{"type":"text","text":"Logger and network inspector for Apple platforms."}],"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse","type":"topic"},"doc://com.github.pulse/documentation/Pulse/RemoteLogger":{"kind":"symbol","type":"topic","fragments":[{"text":"class","kind":"keyword"},{"text":" ","kind":"text"},{"text":"RemoteLogger","kind":"identifier"}],"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger","title":"RemoteLogger","role":"symbol","url":"\/documentation\/pulse\/remotelogger","abstract":[{"type":"text","text":"Connects to the remote server and sends logs remotely. In the current version,"},{"type":"text","text":" "},{"text":"a server is a Pulse Pro app for macOS).","type":"text"}],"navigatorTitle":[{"text":"RemoteLogger","kind":"identifier"}]},"doc://com.github.pulse/documentation/Pulse/RemoteLogger/ConnectionError/errorDescription":{"title":"errorDescription","type":"topic","url":"\/documentation\/pulse\/remotelogger\/connectionerror\/errordescription","identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError\/errorDescription","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"role":"symbol","kind":"symbol","abstract":[]},"doc://com.github.pulse/documentation/Pulse/RemoteLogger/ConnectionError":{"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError","title":"RemoteLogger.ConnectionError","abstract":[],"role":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ConnectionError"}],"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"ConnectionError"}],"kind":"symbol","url":"\/documentation\/pulse\/remotelogger\/connectionerror"}}}